packages/components/checkbox/src/checkbox.css

```diff
@@ -0,0 +1,86 @@
+/* General........................................................ */
+
+.checkbox-container {
+	display: block;
+	position: relative;
+	padding-left: var(--ts-unit-plus);
+	margin: var(--ts-unit-half);
+	cursor: pointer;
+	font-size: var(--ts-fontsize);
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
+	&:hover {
+		& span {
+			background-color: var(--ts-color-gray-lighter);
+		}
+	}
+	& input {
+		position: absolute;
+		opacity: 0;
+		cursor: pointer;
+		height: 0;
+		width: 0;
+	}
+	& span {
+		position: absolute;
+		top: 0;
+		left: 0;
+		height: var(--ts-unit);
+		width: var(--ts-unit);
+		border: var(--ts-border);
+		&:after {
+			content: '';
+			position: absolute;
+			display: none;
+			left: var(--ts-unit-quarter);
+			top: 1px;
```

Comment:
Is there any other solution for this? If not can you provide some comment about the reason

Reply:
I have to make the check mark looks nice.